### PR TITLE
Gha 253 windows

### PR DESCRIPTION
**Pull request by jkeenan**
For: https://github.com/atoomic/perl/issues/253

### CODE REVIEW

File: win32/bin/search.pl [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #!/usr/local/bin/perl -w
-'di';
-'ig00';
```
Comment (atoomic) [inline win32/bin/search.pl:3]: I'm not sure what is the value of these two isolated strings?
Are they required? testing something? was it just an internal joke that we can clear?

File: win32/bin/pl2bat.pl [hunk 2]
```diff
@@ -85,8 +85,8 @@ sub process {
     my $start= $Config{startperl};
     $start= "#!perl"   unless  $start =~ /^#!.*perl/;
     open( FILE, '<', $file ) or die "$0: Can't open $file: $!";
-    @file = <FILE>;
-    foreach $line ( @file ) {
+    my @file = <FILE>;
+    foreach my $line ( @file ) {
```
Comment (atoomic) [inline win32/bin/pl2bat.pl:89]: looks good

**Review by atoomic — APPROVED** [review]
question for `win32/bin/search.pl` otherwise LGTM

**Comment by jkeenan** [comment]
> question for `win32/bin/search.pl` otherwise LGTM

`win32/bin/search.pl` was added to the core distribution by Tim Bunce in d444a431722 in 1997.  The inline comments suggest that it was an adaptation of a program written earlier by Jeffrey Friedl in 1994.  The commit message doesn't say what those two lines might mean.

In general, I think anomalies like this one are best queried on p5p.  Our challenge right now is strict-by-default.  The p.r. adapts those lines to this goal.

Thank you very much.
Jim Keenan

**Pull request merged**